src/arcam/fmj/__init__.py

```diff
@@ -696,6 +700,22 @@ class DecodeModeMCH(IntOrTypeEnum):
         SourceCodes.GAME: bytes([16, 97]),
         SourceCodes.ARC_ERC: bytes([16, 125])
     },
+    (ApiModel.APIST_SERIES, 1): {
+        SourceCodes.DIG1: bytes([21, 94]),
+        SourceCodes.DIG2: bytes([21, 98]),
+        SourceCodes.DIG3: bytes([21, 27]),
+        SourceCodes.DIG4: bytes([21, 27]),
+        SourceCodes.USB: bytes([21, 93]),
+        SourceCodes.NET: bytes([21, 92])
+    },
+    (ApiModel.APIST_SERIES, 2): {
+        SourceCodes.DIG1: bytes([21, 94]),
+        SourceCodes.DIG2: bytes([21, 98]),
+        SourceCodes.DIG3: bytes([21, 27]),
+        SourceCodes.DIG4: bytes([21, 27]),
+        SourceCodes.USB: bytes([21, 93]),
+        SourceCodes.NET: bytes([21, 92])
+    },
```

Comment:
This looks wrong. If they device doesn't support multiple zones, it should at least not list sources for zone 2.

Reply:
I'm not really sure if device supports multiple zones. Do you know if I can spot that from docs? 
I will have a look again and remove it. Also I saw in HA that a 2nd device with name Zone 2 is created but is not usable I think.

Reply:
If it does support it it, it should have different rc codes for the zones.

Reply:
I see this in document

```quote
Zone numbers
The following zone numbers are defined:
• 0x01 – Zone number 1. (Zone 1 is the master zone. Commands that appear zone-less refer to the master zone)
• 0x02 – Zone number 2
```

Reply:
The if there are no distinct RC codes, it wont be able to set the source directly. Ie changing source on zone 2 will affect zone one. Normally there are distinct rc codes per zone.

Reply:
I'm a bit confused, sorry, so you suggest to remove the mapping for Zone 2?

Reply:
Yes

Reply:
removed as suggested!